tests/runtime/int.ts

```diff
@@ -0,0 +1,56 @@
+export function toBytesLE(value: bigint): Uint8Array {
```

Comment:
Would be great to have unit tests for all these functions as well.

Reply:
My logic was that if the runtime code was wrong, then the actual unit tests for the mappings would also fail, so they get implicitly tested.

I agree that if we want to split this into its own package, then unit testing this would be super useful.